engine/team-city-build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+script="$(cd "${0%/*}" 2>/dev/null; echo "$PWD"/"${0##*/}")"
```

Comment:
http://stackoverflow.com/a/673940/4877698 says you're right - lower case for script-local vars